apps/io-services-cms-webapp/src/watchers/on-service-publication-change.ts

```diff
@@ -16,18 +16,22 @@ type OnReleaseActions = RequestHistoricizationAction;
 
 const noAction = {};
 
-const onReleaseHandler = (
-  item: ServicePublication.ItemType
-): RequestHistoricizationAction => ({
+const onReleaseHandler = ({
+  _ts,
+  _etag,
+  ...otherProps
+}: ServicePublication.CosmosResource): RequestHistoricizationAction => ({
   requestHistoricization: {
-    ...item,
-    last_update:
-      item.last_update ?? (new Date().toISOString() as NonEmptyString), // last_update fallback (value is always set by persistence layer) TODO add log
+    ...otherProps,
+    // eslint-disable-next-line no-underscore-dangle
+    last_update: new Date(_ts * 1000).toISOString() as NonEmptyString, // last_update on service-history record corresponds to the _ts on the service-publication record
+    // eslint-disable-next-line no-underscore-dangle
+    version: _etag as NonEmptyString, // version on service-history record corresponds to the _etag on the service-publication record
```

Comment:
**suggestion (non-blocking):** avoid code duplication.

These lines are the same as [on-service-lifecycle-change.ts#onAnyChangesHandler](https://github.com/pagopa/io-services-cms/pull/935/files#diff-31e3e4997d71ad64bb849e0f1ad117d48e12b463d9408a30f93b150a2e975d76R44)...why not centralize this logic in [apps/io-services-cms-webapp/src/historicizer/request-historicization-handler.ts#toServiceHistory](https://github.com/pagopa/io-services-cms/pull/935/files#diff-8d20c959284cf903aa267213021b75fbc82b715b6ea1f2eb29ae7150b14bd689R24)?

Reply:
e7143dee0696d567cb39874ff987014daf0a275f